### PR TITLE
Route view

### PR DESCRIPTION
**Pull request by JosefForkman**
- add tanstack router
- add /view route for the add new modal button

### CODE REVIEW

**Comment by sebastianandreasson** [comment]
The logic is flipped so /view shows the button and / hides it.

I would either rename the route to /edit or flip the logic.

**Comment by sebastianandreasson** [comment]
pocketbase.exe slipped through gitignore, should be removed

File: src/routes/view.lazy.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { createLazyFileRoute } from '@tanstack/react-router'
+import Header from '../components/Header'
+import Events from '../components/Events'
+import AddEventButton from '../components/AddEventButton/AddEventButton'
+
+export const Route = createLazyFileRoute('/view')({
+  component: About,
+})
+
+function About() {
```
Comment (sebastianandreasson) [inline src/routes/view.lazy.tsx]: misleading name about

**Comment by sebastianandreasson** [comment]
Looks good, I'm not sure if I'm a fan of tanstack router because it seems overly complex but maybe its good for a larger project. Just fix the review comment regarding naming of the "About" component and its good to merge

**Comment by JosefForkman** [comment]
I think everything has been fixed. 
I changed route to /edit and removed pocketbase.exe

**Pull request merged**